suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -22,23 +22,45 @@ import { TransactionListItemIcon } from './TransactionListItemIcon';
 
 type TransactionListItemProps = {
     transaction: WalletAccountTransaction;
+    isFirst?: boolean;
+    isLast?: boolean;
 };
 
 type TransactionTypeProperties = {
-    prefix: 'From' | 'To';
-    sign: '+' | '-';
-    signColor: keyof Colors;
+    prefix: string;
+    sign?: string;
+    signColor?: Color;
 };
-const transactionTypePropertiesMap: Partial<Record<TransactionType, TransactionTypeProperties>> = {
+const transactionTypePropertiesMap = {
     recv: { prefix: 'From', sign: '+', signColor: 'green' },
-    sent: { prefix: 'To', sign: '-', signColor: 'red' },
-};
+    sent: { prefix: 'To', sign: undefined, signColor: 'red' },
+    self: { prefix: 'Self', sign: undefined, signColor: undefined },
+    joint: { prefix: 'Joint', sign: undefined, signColor: undefined },
+    failed: { prefix: 'Failed', sign: undefined, signColor: undefined },
+    unknown: { prefix: 'Unknown', sign: undefined, signColor: undefined },
+} as const satisfies Partial<Record<TransactionType, TransactionTypeProperties>>;
 
-const transactionListItemStyle = prepareNativeStyle(_ => ({
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-}));
+type transactionListItemStyleProps = {
+    isFirst: boolean;
+    isLast: boolean;
+};
+const transactionListItemStyle = prepareNativeStyle<transactionListItemStyleProps>(
+    (utils, { isFirst, isLast }) => ({
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        backgroundColor: utils.colors.gray0,
+        marginHorizontal: utils.spacings.small,
+        marginBottom: isLast ? utils.spacings.small : 0,
```

Comment:
if case you need such a lot of if's use `extend` instead, check component `suite-native/atoms/src/NumPadButton.tsx` for example how to use it

Reply:
fixed in 871ceb966